### DISCUSSION TITLE
unable to authenticate push

### DISCUSSION THREAD

**Discussion by LucaMag1** [community/community#30386] [Other Feature Feedback, Questions, & Ideas]
Im trying to push a commit and its asking me to complete authentication in browser. 
A pop up appeared and it asked for my password which i had to change because i forgot it. 
After i changed my password the pop up disappeared and now i cant complete the authentication to push the commit. Can someone please help me solve this?

**Comment by airtower-luna** [comment]
What happens if you try to push now? It's best if you share the full command and output.

**Reply by LucaMag1** [reply]
i typed git push and it says: " info: please complete authentication in your browser... "

**Reply by airtower-luna** [reply]
Hm. I'm not familiar with the browser authentication process from Git, but it looks like [Git Credential Manager](https://github.com/GitCredentialManager/git-credential-manager) didn't notice the process was interrupted. So what I'd try is to check your local credential manager and the authorized OAuth applications on GitHub, and clean up any half-done credentials. Be careful not to delete anything you want to keep, though. :sweat_smile: 

Alternatively you could disable the credential manager, but then you'd have to manually create and provide a PAT.

**Reply by LucaMag1** [reply]
i checked the authorized OAuth applications however it says "no authorized applications", and my local credential manager doesn't have anything related to github.

**Reply by airtower-luna** [reply]
That's unfortunate, I hope someone with deeper knowledge of the credential manager can chip in. Or that the authentication attempt times out eventually. :sweat_smile:

**Reply by LucaMag1** [reply]
thanks for your help
